src/bp/nlu/stan/validation/validate.ts

```diff
@@ -49,8 +41,8 @@ const makeIntentChecker = (contexts: string[]) => (
   intent.slots.forEach(variableChecker)
 }
 
-export async function validateTrainInput(rawInput: any): Promise<TrainInput> {
-  const validatedInput: TrainInput = await validate(rawInput, TrainInputSchema, {})
+export async function validateTrainInput(rawInput: any): Promise<http.TrainRequestBody> {
```

Comment:
What happens when these functions fail validation? does it simply return `{}` ?

Reply:
it throws